### ISSUE TITLE
CSV file used in training_script.py

### ISSUE THREAD

**Issue by richmondu** [ee09115/spoofing_detection#5]
Can you provide the CSV file training data used for training_script.py? 
joblib could not load the classifiers in Raspberry Pi 3B+ due to scikit-learn incompatible versions. 
I need to use scikit-learn 0.20.2 instead of 0.19.1 so I need to train the classifier with this version.

Thank you!

**Comment by ifcoxs** [comment]
![image](https://user-images.githubusercontent.com/32489220/54021433-1d76c400-41cb-11e9-8d96-3d0fa9701dd9.png)
Should it be like this?Looking forward to the author's answer

**Comment by ee09115** [comment]
Hello,

I am not allowed to share the .csv files to train the classifiers. These public databases should be downloaded from the corresponding providers (e.g.: [Replay-Attack](https://www.idiap.ch/dataset/replayattack), [Print-Attack](https://www.idiap.ch/dataset/printattack)).

@ifcoxs Yes it should be like that.

Regards,

**Comment by lunana123** [comment]
> ![图片](https://user-images.githubusercontent.com/32489220/54021433-1d76c400-41cb-11e9-8d96-3d0fa9701dd9.png)
> 应该是这样吗？期待作者的回答
你好，请问你的csv文件是从哪里下载的，可以分享一下吗

**Comment by ifcoxs** [comment]
@lunana123 这个是用数据集生成的文件。通过main.py中获取摄像头的图像，然后提取特征的方法，可以推出作者提取了哪些特征向量以及提取方式；通过training_script.py中load_data(path_name, n_rows, n_cols)函数可以推出他是如何组织csv文件，然后送进分类器进行训练的。
我是这么理解的！

**Comment by DecentMakeover** [comment]
@lunana123, @ifcoxs  can you tell the difference between the label(y_train) and pai_id(PAI_ID).

Thanks

**Comment by bcba25** [comment]
@ee09115 since you are not allowed to share csv, could you explain on how to generate our own ones please ? Or at least give a fake sample of what should be placed there
